x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/privilege_escalation_unusual_printspooler_childprocess.json

```diff
@@ -50,5 +50,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 4
+  "version": 5
```

Comment:
Version bump occurred in last `8.2 Rules` PR: https://github.com/elastic/kibana/pull/129526/files#diff-d5b20e6da8c51ab3d8d52bf3e6006c2e4cf089ac8108c799e9793131cb7da3baR53